docs/DEVELOPING.md

```diff
@@ -343,6 +343,139 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## Oracle Database Connector
+
+The [Elastic Oracle connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/oracle.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
```

Comment:
Since we had a discussion with @danajuratoni for `network drive` and `Amazon S3` to point to `8.6` branch and now `gcs`, `abs` and `generic database` is part of `8.7` release we pointing to `8.7` branch.

Reply:
As per the previous discussion for `network drive` and `S3`. We point to a specific branch.

Reply:
Main reason is to not leave the branch with the connector client when clicking the link.
This was clarified in #549 (comment)
and I'm good to use Chenhui's suggestion as that is more flexible as still keeps the users in the selected branch.

Reply:
Replace with a relative link.